### ISSUE TITLE
Counter is gone since latest MagicMirror update (low priority, screen off still works)

### ISSUE THREAD

**Issue by hweigel** [bugsounet/MMM-NewPIR#23]
Ever since the update notification popped today (2020/01/04) that MagicMirror is 161 commits behind, I ran git pull (of course followed by npm i), then rebooted, 

But whatever I try, the counter after "Auto Turn Off Screen:" isn't showing anymore. Functionality remains, even on my rpi4 (see my comments in #10 for further details).

PS: npm run config:check throws a buttload of rule errors since the latest MM update (which I disabled with `/* eslint-disable */` at the top of my config.js).

**Comment by bugsounet** [comment]
hi, MagicMirror v2.11.0 need new libraries.
so retry new installation for matching with new libraries

```sh
cd ~/MagicMirror/modules
rm -rf MMM-NewPIR
git clone https://github.com/bugsounet/MMM-NewPIR.git
cd MMM-NewPIR
npm install
```
if no error when installing
just restart your mirror

**Comment by hweigel** [comment]
Hey Cedric, Switching RPI4 didn't do anything (even after rebooting).

FYI, wiping the NewPIR directory and reinstalling threw those warnings:

```
npm WARN deprecated mkdirp@0.5.4: Legacy versions of mkdirp are no longer supported. Please update to mkdirp 1.x. (Note that the API surface has changed to use Promises in 1.x.)
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
```
Unfortunately, even after multiple reboots, and testing all possible variations of `force:false/true` and `rpi4:false/true`, the counter remains gone :-(.

**Comment by bugsounet** [comment]
give your MMM-NewPIR config plz ?

**Comment by hweigel** [comment]
...Aaand the counter is **back**!

 Since you asked for the configuration, I wondered what would happen if I moved the counter to another region. Did it and: boom - counter back! Well, kind of, at least I saw it.

Before the MM update today, both the text and counter would get the formatting from `#NEWPIR_TEXT` but didn't inherit any settings from `#NEWPIR` (same behaviour as in e.g. MMM-Wallpaper).

Now, the counter takes its formatting from '#NEWPIR', and the text block inherits these settings (I guess that's what you wanted all along).  

Long story short, I moved all CSS settings from #NEWPIR_TEXT to #NEWPIR, and everything is fine again. Well, except for the virus, which still sucks.

Anyhoo, sorry for the bother. Stay safe Cedric!

/Kirk out
